test/ucp/storetest/shared.go

```diff
@@ -37,13 +37,15 @@ const (
 	ResourcePath2       = "System.Resources/resourceType2/resource2"
 	ResourcePath3       = "System.Resources/resourceType2/Resource3"
 	NestedResourcePath1 = "System.Resources/resourceType1/resource1/nestedType/nested1"
+	NestedResourcePath2 = "System.Resources/resourceType1/resource1/nestedType/nested2"
+	NestedResourcePath3 = "System.Resources/resourceType1/resource2/nestedType/nested3"
+	NestedResourcePath4 = "System.Resources/resourceType1/resource2/nestedType/nested4"
```

Comment:
Now that queries are limited to a single type, I needed to add some more resources to make sure we're testing the same logic.